sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/OfferEvaluator.java

```diff
@@ -436,4 +438,30 @@ private static void logOutcome(StringBuilder stringBuilder, EvaluationOutcome ou
 
         return null;
     }
+
+    @VisibleForTesting
+    UUID getTargetConfig(PodInstanceRequirement podInstanceRequirement, Collection<Protos.TaskInfo> taskInfos) {
+        if (podInstanceRequirement.getRecoveryType().equals(RecoveryType.NONE) || taskInfos.isEmpty()) {
```

Comment:
Correct.  PodInstanceRequirements [default to NONE as recoveryType](https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/PodInstanceRequirement.java#L98).  When type is `NONE`, that's the initial launch condition.